TKGame/Enemy/KnightEnemy.cs

```diff
@@ -54,12 +30,12 @@ public override void Update(GameTime gameTime)
                 if (Position.X > playerPosition.X)
                 {
                     Position.X -= velocity.X;
-                    Orientation = SpriteEffects.FlipHorizontally;
+                    Orientation = SpriteEffects.None;
                 }
                 if (Position.X < playerPosition.X)
                 {
                     Position.X += velocity.X;
-                    Orientation = SpriteEffects.None;
+                    Orientation = SpriteEffects.FlipHorizontally;
                 }
                 if (Position.Y > playerPosition.Y)
                 {
```

Comment:
This stuff needs to moved into a component. See Player components, this would probably fall under physics and input.